### ISSUE TITLE
Using searchFeature with TileWMS layer?

### ISSUE THREAD

**Issue by borjaevo** [Viglino/ol-ext#889]
I couldn't find what kind of layers are compatible with searchFeature component in the docs. Also i have many doubts about how to specify the property of the layer to look for. 
In the docs it appears that the property is "property to display in the index, default 'name'. I understand "name" is the name of the feature. In my case, this would be "Address". 
Reading the code of the example the same property value is assigned to 
`property: $(".options select").val()`

i dont understand how this is related to a feature of the Vector layer 'Fonds de guerre 14-18' of this example or is taking default "name" value some way. 
How can i use "Address" feature of my  TileWMS layer to search in?
Thanks

**Comment by Viglino** [comment]
SearchFeature is a control to search [ol/Feature](https://openlayers.org/en/latest/apidoc/module-ol_Feature-Feature.html) ie. vector feature (you don't have features in a raster layer to search in).

It search a property value and the property name must be passed to the control to know what.

In the example it search features in a GeoJSON and look for a string in the feature properties (commune, region or text).
Basically it test `feature.get('text')` contains `string`

**Comment by borjaevo** [comment]
Thank you very much. We have it working now!
